packs/feats/halfling-luck.json

```diff
@@ -28,7 +28,33 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
+                "outcome": [
+                    "failure",
+                    "criticalFailure"
+                ],
```

Comment:
```suggestion
                "outcome": [
                    "failure"
                ],
```
This triggers on failure, not critical failure.

Reply:
> Trigger You fail a skill check or saving throw.

I believe the RAW is that critical failures are a subset of failure that without a distinction excepting them also apply to things like Halfling Luck. 

From page 8 of Player Core
> Once a check is rolled, the GM compares the result to a
> target number called the difficulty class (DC) to determine
> the outcome. If the result of the check is equal to or greater
> than the DC, the check is successful. If it is less, the check is
> a failure.

I would expect if critical failure were meant to be excluded it would be like Dubious Knowledge and other examples that specify _When you fail (but don’t critically fail)_

Probably the biggest indicator that this is intended are these two feats as the 2nd one makes it clear that the first one (which uses the same language as Halfling Luck) works on fail and critical fail.

![image](https://github.com/foundryvtt/pf2e/assets/99048395/cc5e5945-e155-4bf3-ad78-48bb5d7afd49)
![image](https://github.com/foundryvtt/pf2e/assets/99048395/5078097c-6dee-4dab-99d2-360d486088af)